src/web/app/pages-help/instructor-help-page/example-box/example-box.component.scss

```diff
@@ -0,0 +1,62 @@
+.bs-example {
+  background-color: #FAFAFA;
+  border-color: #E5E5E5 #EEE #EEE;
+  border-style: solid;
+  border-width: 1px 0;
+  -moz-box-shadow: inset 0 3px 6px rgba(0, 0, 0, .05);
+  -webkit-box-shadow: inset 0 3px 6px rgba(0, 0, 0, .05);
```

Comment:
You shouldn't need this. Angular CLI has autoprefixer built-in. Similarly for other vendor-prefixed style items.